geode-pulse/src/test/java/org/apache/geode/tools/pulse/tests/rules/ScreenshotOnFailureRule.java

```diff
@@ -45,7 +45,9 @@ private void takeScreenshot(String screenshotName) {
     if (driver instanceof TakesScreenshot) {
       File tempFile = ((TakesScreenshot) driver).getScreenshotAs(OutputType.FILE);
       try {
-        FileUtils.copyFile(tempFile, new File("build/screenshots/" + screenshotName + ".png"));
+        File screenshot = new File("build/screenshots/" + screenshotName + ".png");
+        FileUtils.copyFile(tempFile, screenshot);
+        System.err.println("Screenshot saved to: " + screenshot.getCanonicalPath());
```

Comment:
This is intentional, it adds some output to help find where the screenshots end up when one gets taken.